openshift-hack/e2e/annotate/generated/zz_generated.annotations.go

```diff
@@ -2067,6 +2067,10 @@ var Annotations = map[string]string{
 
 	"[sig-node] [Feature:Example] Secret should create a pod that reads a secret": " [Suite:openshift/conformance/parallel] [Suite:k8s]",
 
+	"[sig-node] [Feature:GPUDevicePlugin] Sanity test for Nvidia Device should run gpu based matrix multiplication": " [Disabled:SpecialConfig] [Suite:k8s]",
+
+	"[sig-node] [Feature:GPUDevicePlugin] Sanity test for Nvidia Device should run nvidia-smi cli": " [Disabled:SpecialConfig] [Suite:k8s]",
+
```

Comment:
Please make sure these tests won't break our CI. They might need different images and that will require us to mirror them. 

If we have any issues, we should disable them.